### PR TITLE
Introduce action to increase dovecot DH parameters size

### PR DESCRIPTION
**Pull request by SandakovMM**
Originally required by centos2alma since we upgrade rouncube during the converion process

### CODE REVIEW

File: pleskdistup/actions/emails.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import json
+import subprocess
+
+from pleskdistup.common import action, log, util
+
+
+class IncreaseDovecotDHParameters(action.ActiveAction):
```
Comment (kpushkaryov) [inline pleskdistup/actions/emails.py:8]: For the sake of generality I propose to call that like `SetMinDovecotDhParamSize` and make `dhparam_size` configurable.

File: pleskdistup/actions/emails.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    def _is_required(self) -> bool:
+        proc = subprocess.run(
+            ["/usr/sbin/plesk", "sbin", "sslmng", "--show-config"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+        )
+        if proc.returncode != 0:
+            log.warn(f"Failed to get ssl configuration by plesk sslmng: {proc.stdout}\n{proc.stderr}")
+            return False
```
Comment (kpushkaryov) [inline pleskdistup/actions/emails.py:22]: Do we really want to skip it in case of failure?

File: pleskdistup/actions/emails.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import json
+import subprocess
+
+from pleskdistup.common import action, log, util
+
+
+class IncreaseDovecotDHParameters(action.ActiveAction):
+    def __init__(self):
+        self.name = "increase Dovecot DH parameters size to 2048 bits"
+        self.dhparam_size = 2048
```
Comment (kpushkaryov) [inline pleskdistup/actions/emails.py:11]: Please declare `dhparam_size` type.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

File: pleskdistup/actions/emails.py (hunk 2 above)
Comment (kpushkaryov) [reply pleskdistup/actions/emails.py:24]: What about that?

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

File: pleskdistup/actions/emails.py (hunk 2 above)
Comment (SandakovMM) [reply pleskdistup/actions/emails.py:24]: Previous commentary sucked as pending. Please take a look

**Review by kpushkaryov — APPROVED** [review]

**Comment by kpushkaryov** [comment]
Please rebase before merge.

**Pull request merged**